build/automation/lib/aws.mk

```diff
@@ -325,6 +325,13 @@ aws-ecr-get-image-digest: ### Get ECR image digest by matching tag pattern - man
 			--repository-name $(shell echo $(REPO) | sed "s;$(AWS_ECR)/;;g") \
 	" | make -s docker-run-tools CMD="jq -rf $$file" | head -n 1
 
+aws-ecr-untag: ### Untag (and remove if none) images in the ECR - mandatory: NAME=[repository name],TAG=[string to match tag of an image] optional:INCLUDE_LATEST=[true,false]
```

Comment:
Make target name says `untag` but the AWS cli command mention batch deleting the image, is the target name correct?
(Highlighted in Github Maintainers meeting)

Reply:
Yes it's the command that effectively untags, and therefore deletes if no tags exist (hence the mention of that in the help text above)

Reply:
I think the text need to be a bit clearer that it will delete the image, don't think some using an untag target would expect it to delete the image